client/client.go

```diff
@@ -40,8 +40,8 @@ type Client interface {
 	Txn() (kv.TxnStore, error)
 
 	// Store returns access to the distributed configuration store with a namespace.
-	Store(namespace string) (kv.Store, error)
+	Store(opts kv.Options) (kv.Store, error)
```

Comment:
How would you feel about instead of passing `kv.Options` in we created a `Prefixer` interface which looks like:

```
type Prefixer interface {
    Prefix() string
}
```

Then we can have a standard implementation which puts the zone, environment, and namespace together, but can also supports other kinds of prefixes where appropriate?

Reply:
I'm hesitate to go down that route unless we have to, cause we use zone to create cluster rather than putting zone in the key. 
Unless one day we start to look at the prefix (or namespace) and find the cluster for that prefix and then look up the key in the cluster, we can't get away with only a prefix provided.